src/GrpcProtos/capture.proto

```diff
@@ -308,5 +322,6 @@ message ProducerCaptureEvent {
     SchedulingSlice scheduling_slice = 12;
     ThreadName thread_name = 13;
     ThreadStateSlice thread_state_slice = 14;
+    SystemMemoryUsage system_memory_usage = 15;
```

Comment:
Move up to keep the alphabetical order.

Reply:
I forgot to change the order after the renaming! Thanks so much for pointing out this!